install.sh

```diff
@@ -77,88 +76,172 @@ ros_git_get() {
 	done
 	if $NEEDS_INSTALL; then
 		instlog "Installing $INSTALL_URL in $CATKIN_DIR/src"
-		git clone -q $INSTALL_URL --depth=1 --recursive
+		git clone -q $INSTALL_URL --depth=100 --recursive
 	fi
 }
 
 
+#=======================#
+# Configurable Defaults #
+#=======================#
+
+CATKIN_DIR=~/mil_ws
+BASHRC_FILE=~/.bashrc
+
+
 #======================#
 # Script Configuration #
 #======================#
 
-# Sane installation defaults for no argument cases
-REQUIRED_OS="trusty"
-CATKIN_DIR=~/mil_ws
-INSTALL_ALL=false
+# Install no project by default, the user must select one
 INSTALL_SUB=false
+INSTALL_PRO=false
```

Comment:
Yes, it costs 1 Bitcoin, I will be accepting payment to this address: 1HHbkF94QT2WjsM2bqZoGbR2mXEmTN